common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OAuth2TokenCache.java

```diff
@@ -54,4 +53,12 @@ public OAuth2TokenCache(Context context) {
     public abstract void saveTokens(final T oAuth2Strategy,
                                     final U request,
                                     final V response) throws ClientException;
+
+    protected Context getContext() {
+        return mContext;
+    }
+
+    protected void setContext(Context context) {
```

Comment:
Same here. Shouldn't be part of the `OAuth2TokenCache` API

Reply:
See above.

Reply:
Let's remove this and declare `private final`